Userland/Libraries/LibGUI/Scrollbar.cpp

```diff
@@ -174,6 +174,28 @@ void Scrollbar::paint_event(PaintEvent& event)
         hovered_component_for_painting = Component::None;
 
     painter.fill_rect_with_dither_pattern(rect(), palette().button().lightened(1.3f), palette().button());
+    if (gutter_click_state != GutterClickState::NotPressed && has_scrubber() && hovered_component_for_painting == Component::Gutter) {
+        VERIFY(!scrubber_rect().is_null());
+        Gfx::IntRect rect_to_fill = rect();
+        if (orientation() == Orientation::Vertical) {
+            if (gutter_click_state == GutterClickState::Before) {
+                rect_to_fill.set_top(decrement_button_rect().bottom());
+                rect_to_fill.set_bottom(scrubber_rect().top());
+            } else {
```

Comment:
I think making this a else if is more expressive, same below.
It's clear what state is left in gutter_click_state by reading the code around it, but I prefer just reading a single line to understand.

Reply:
Maybe i could just put a `VERIFY(gutter_click_state == GutterClickState::After);` there, to make sure of the state, and also catch any issues that might arise if there were any other states introduced?